advent2019/words_advent.py

```diff
@@ -10,19 +10,19 @@
 import random
 
 n = int(input('How many words do you want? '))
-numbers = []
+numbers = set()
```

Comment:
With a set, if two of the random numbers are equal, you won't get the number of words you want.

Reply:
Also it sometimes happens that the indices are different but the words are the same!

Reply:
Let me fix these things...